src/longopt/gmtinfo_inc.h

```diff
@@ -24,10 +24,10 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'A', "range|ranges",
+	{ 0, 'A', "report",
 	          "a,t,s",             "all,per_table,per_segment",
 	          "",                  "" },
-	{ 0, 'C', "per_column",        "", "", "", "" },
+	{ 0, 'C', "columns",        "", "", "", "" },
```

Comment:
The reason a I named it `numeric` in Julia it's because that is what it really does. It returns a numeric vector in contra-position of no -C that return a text struct. And even in CLI  that difference is truly visible.

Reply:
We can do alias with numeric|colums.

Reply:
Alias done.